components/brave_shields/resources/panel/components/main-panel/index.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react'
+
+import * as S from './style'
+import { FabulouslyLargeToggle } from '../../../../../web-components/toggle'
+import AdvancedControlsContent from '../advanced-controls-content'
+import { getLocale } from '../../../../../common/locale'
+
+function MainPanel () {
+  const [isExpanded, setIsExpanded] = React.useState(false)
+
+  return (
+    <S.Box>
+      <S.PanelHeader>
+        <S.SiteTitle>brave.com</S.SiteTitle>
+      </S.PanelHeader>
+      <S.ToggleBox>
+        <FabulouslyLargeToggle
+          brand='shields'
+          accessibleLabel={getLocale('braveShieldsEnable')}
+        />
+        <S.StatusText>
+          {getLocale('braveShields')}
+          {' '}
+          <span>{getLocale('braveShieldsUp').toUpperCase()}</span>
```

Comment:
Should this be a single string so that the syntax order of the UP and DOWN relative to the rest of the sentence makes sense for the specific language?

Reply:
done